statsmodels/tsa/vector_ar/irf.py

```diff
@@ -95,21 +94,19 @@ def plot(self, orth=False, impulse=None, response=None,
             variable affected by the impulse
         signif : float (0 < signif < 1)
             Significance level for error bars, defaults to 95% CI
-        subplot_params : dict
-            To pass to subplot plotting funcions. Example: if fonts are too big,
-            pass {'fontsize' : 8} or some number to your taste.
-        plot_params : dict
-
+        subplot_kwargs : dict
+            To pass to matplotlib.add_subplot
         plot_stderr: bool, default True
             Plot standard impulse response error bands
         stderr_type: string
             'asym': default, computes asymptotic standard errors
             'mc': monte carlo standard errors (use rpl)
         repl: int, default 1000
```

Comment:
should have a `n` in front ?
(that's what I usually use, do we have a standard for Monte Carlo names?)